src/cycleclock.h

```diff
@@ -210,7 +210,7 @@ inline BENCHMARK_ALWAYS_INLINE int64_t Now() {
 #else
   uint64_t cycles;
   asm volatile("rdtime %0" : "=r"(cycles));
-  return cycles;
+  return static_cast<int64_t>(cycles);
```

Comment:
maybe we should check the range before the cast?

Reply:
We don't do that anywhere else though. I'm not sure what we would even do in that situation?

Reply:
assert i guess?  the C++ style guide talks about this a bit in discussing avoidance of unsigned types ("In particular, do not use unsigned types to say a number will never be negative. Instead, use assertions for this.")

but i agree we don't currently do it anywhere else.  happy to punt this to another PR.